lib/output.js

```diff
@@ -597,6 +598,12 @@ function OutputStream(options) {
             return true;
         }
 
+        if (output.option('webkit')
+            && output.parent(0) instanceof AST_PropAccess
+            && output.parent(1) instanceof AST_Assign) {
```

Comment:
> In any case the patch above does not look at the parent  expression  leading to unnecessary parens:

Indeed - fixed in https://github.com/mishoo/UglifyJS2/pull/2056/commits/247014538a1920cc90a23fa412de431dabaf4cb1